content/id/docs/tutorials/security/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Keamanan"
+weight: 40
+---
+
+Keamanan adalah hal yang sangat penting bagi kebanyakan organisasi dan individu yang menjalankan kluster Kubernetes. Anda dapat menemukan [daftar periksa keamanan](/docs/concepts/security/security-checklist/) dasar di tempat lain dalam dokumentasi Kubernetes.
```

Comment:
Small comment... "kluster" must be "klaster"

Reply:
Updated in the next commit kak @ariscahyadi